packages/big-design-patterns/src/components/Page/styled.tsx

```diff
@@ -0,0 +1,44 @@
+import { Flex } from '@bigcommerce/big-design';
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css, CSSProperties } from 'styled-components';
+
+export interface Background {
+  src: string;
+  backgroundSize?: CSSProperties['backgroundSize'];
+  backgroundPosition?: CSSProperties['backgroundPosition'];
+  backgroundRepeat?: CSSProperties['backgroundRepeat'];
+}
+
+export const StyledPageBackground = styled.div.attrs({ theme: defaultTheme })<{
+  background?: Background;
+}>`
```

Comment:
Based on the way `background` is used in `PageBackground`, it looks like it'll always be defined 🤔 do we need the `if (background) {` in this component?